projects/angular/src/forms/datepicker/day.ts

```diff
@@ -50,7 +51,7 @@ export class ClrDay {
   @Input('clrDayView')
   public set dayView(day: DayViewModel) {
     this._dayView = day;
-    this.dayString = this._dayView.dayModel.toDateString();
+    this.dayString = `${this._dayView.dayModel.toDateString()} ${this.dayView.isSelected ? this.commonStrings.keys.datepickerSelectedLabel : ''}`;
```

Comment:
You can't just concatenate like this, since not every language uses this structure. You'll need to put a placeholder in the string.

I need to run this by our A11Y team before we can accept it

Reply:
Technically, this will add a trailing space to the end of the day if it's not selected. I doubt it will affect screen readers, but if we wanted to be more correct we could do something like:

```
const selectedText = this.dayView.isSelected ? ` ${this.commonStrings.keys.datepickerSelectedLabel}` : '';
this.dayString = `${this._dayView.dayModel.toDateString()}${selectedText}`
```

Reply:
We cannot use concatenation though, like Ashley said. We need a placeholder in the common string.